velox/vector/tests/VectorTest.cpp

```diff
@@ -1684,6 +1684,52 @@ TEST_F(VectorTest, multipleDictionariesOverLazy) {
   }
 }
 
+TEST_F(VectorTest, dictionaryResize) {
```

Comment:
This test is a bit hard to follow. Would you clarify what it does?

Reply:
I have modified this and added a few more tests.